templates/etc/supervisor/conf.d/torquedata.conf

```diff
@@ -0,0 +1,9 @@
+[program:torquedata]
+directory=/data/www/torque/torquedata
+user:vagrant
+command=pipenv run flask run
+autostart=true
+autorestart=false
+redirect_stderr=true
+redirect_stdout=true
+environment=FLASK_APP=torquedata
```

Comment:
The usual :-).

Reply:
Time to update my default .editorconfig 😂